jupyterlab_gcedetails/src/details_widget.tsx

```diff
@@ -115,7 +115,9 @@ export class VmDetails extends React.Component<Props, State> {
       this.setState({ details: details });
       notebookService.projectId = details.project.projectId;
       notebookService.locationId = details.instance.zone;
-      notebookService.instanceName = details.instance.name;
+      const instanceNameSplit = details.instance.name.split('/');
```

Comment:
You can also use the `pop()` method for future cases like this where you want the last piece of a split-string. It mutates the array but in this case you don't care about the other pieces.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop